pkg/driver/controller_modify_volume_test.go

```diff
@@ -64,7 +65,9 @@ func TestParseModifyVolumeParameters(t *testing.T) {
 				ModificationKeyIOPS:       validIops,
 				ModificationKeyThroughput: validThroughput,
 				ModificationAddTag:        validTagSpecificationInput,
+				ModificationAddTag + "_1": "key2={{ .PVCName }}",
```

Comment:
Is there a reason we are only testing modifyVolume interpolation on PVCName? Not PVCNamespace and PVName? 

Is this because we already trust our unit tests for string interpolation elsewhere? I would consider adding. 

![image](https://github.com/user-attachments/assets/f1b66fbe-4867-49ed-8ef4-9f5c1ba02000)